components/web/goaccess/goaccess.p5m

```diff
@@ -22,14 +23,17 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-file path=etc/goaccess.conf preserve=true
+dir path=etc/goaccess
```

Comment:
Directory actions are only required, if no other files below are owned by this package. The directories will be created automatically if needed. Please remove.

Reply:
Thanks, it's removed now.